### ISSUE TITLE
no colons mode for massConvert

### ISSUE THREAD

**Issue by ThePlenkov** [SAP-samples/hana-developer-cli-tool-example#50]
Hi!

Since we've introduced namespace option in #43  
and also have automatic synonyms generation #48 

i thought how to combine those two things. Pretent we have a table like that in the database:
```
com.my_company.flights_model::flights
```

So right now `hana-cli mc` gives us `![com_my_company_flight_model::flights]` entity. We can use it in the next cds like that:
```
using { ![com_my_company_flight_model::flights]  as flights } from './flights'
```

If we use `--namespace` option we can make it simple to import just namespace, but because of `::`  we still have to escape table name later

So what my proposal is - to introduce a special mode when actually the original path will be kept, but `::` symbol will be replaced with one more dot.

So we will have in the generated file
```
entity com.my_company.flights_model.flights {}
```

then we'll be able to use this cds in a very flexible way `using com.my_company.flights_model as model` or `using com.my_company` and so on. Also because we don't have colons we can access table like that `model.flights` in CAP cds.

**Comment by ThePlenkov** [comment]
as a proposal: we can make `--keepPath`  or `--noColons`  flags. Namespace option will still make sense in those cases when we wish to use high-level namespace

**Comment by jung-thomas** [comment]
Merged and delivered with 2.202105.9
